takeshape/takeshape-starter-ecommerce-kitchen-sink
# use indexed payment query

### Test Plan (Steps to test):

1. Login, go to purchases page
1. Notice how quickly purchases show up! Rejoice!
1. Make a new purchase... There may be a delay in it showing on the list. But it should within 1 minute.
